base/atomics.jl

```diff
@@ -29,6 +29,30 @@ typealias IntTypes Union{inttypes...}
 typealias FloatTypes Union{floattypes...}
 typealias AtomicTypes Union{atomictypes...}
 
+doc"""
```

Comment:
What do you mean by "this should drop the `doc` prefix"? Do you mean "please remove the prefix"?